raiden/api/python.py

```diff
@@ -121,11 +123,11 @@ def connect_token_network(
                 channels opened by other participants.
         """
         if not isaddress(token_address):
-            raise InvalidAddress('not an address %s' % pex(token_address))
+            raise InvalidAddress('token_address must be a valid address in binary')
+
         try:
             connection_manager = self.raiden.connection_manager_for_token(token_address)
         except InvalidAddress:
-            # token is not yet registered
```

Comment:
Why remove the comment here?

Reply:
Because `InvalidAddress` exception is used when an address is invalid, it seems enough to me.

Reply:
yes for you and for me it may be obvious but it's still a jump in logic, requiring knowledge of how things works. I think it's quite a beneficial comment. Can you please leave it?